snooty/test_page_data.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from .n import FileId
+from .page_database import PageDatabase
+from .parser import Project
+from .test_project import Backend
+
+
+def test_persistance() -> None:
```

Comment:
Just wondering if this `persistance` is intentionally French 🇫🇷 , or if it should be `persistence`

Reply:
Non. L'orthographe est difficile